source/SAMD21PDM.cpp

```diff
@@ -67,7 +73,6 @@ SAMD21PDM::SAMD21PDM(Pin &sd, Pin &sck, SAMD21DMAC &dma, int sampleRate, int clo
     this->clockRate = clockRate;
     this->sampleRate = sampleRate;
```

Comment:
Is `sampleRate` needed anymore?

Reply:
I think it's useful to set the sample rate and have it accessible through the object since it will be used in most microphone applications. I've just pushed to make the sample rate settable but fix the clock rate to 16*sample rate as anything else wouldn't really make sense.